src/app/components/MostRead/Canonical/Item/index.tsx

```diff
@@ -44,9 +44,28 @@ export const MostReadLink = ({
   href,
   children,
   size,
+  id,
+  position,
   eventTrackingData,
 }: PropsWithChildren<MostReadLinkProps>) => {
-  const clickTrackerHandler = useClickTrackerHandler(eventTrackingData);
+  const positionIndex =
+    typeof position === 'string' ? parseInt(position, 10) : position;
+
+  const eventTrackingDataExtended = {
+    ...eventTrackingData,
+    itemTracker: {
+      ...eventTrackingData?.itemTracker,
+      type: 'most-read-promo',
+      text: title,
+      position: positionIndex,
+      resourceId: id || href,
+      ...(eventTrackingData?.groupTracker?.position && {
+        groupPosition: eventTrackingData.groupTracker.position,
+      }),
```

Comment:
Don't think we need this as it should be on the `groupTracker` level rather than inside the `itemTracker`